src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -255,6 +258,8 @@ protected int doWork() {
         // Setup the inputs
         ////////////////////////////////////////////////////////////////////////
         final LiftOver liftOver = new LiftOver(CHAIN);
+        liftOver.setShouldLogFailedIntervalsBelowThreshold(LOG_FAILED_INTERVALS);
```

Comment:
This is OK but I question the use of 'should' in the API name. To me, 'should' indicates a recommendation or suggestion, when in fact this is a direct toggle switch. E.g. `setLogFailedIntervalsBelowThreshold()`.

Reply:
that "should" have been dealt with in htsjdk...not here. :-/

Reply:
Well, something to think about next time I guess..